### ISSUE TITLE
Mount temp gets shredded?

### ISSUE THREAD

**Issue by astrobokonon** [LowellObservatory/NightWatch#39]
See attached image; not sure what's going on with this one unfortunately.

<img width="1441" alt="Screen Shot 2020-05-11 at 7 37 15 PM" src="https://user-images.githubusercontent.com/1168555/81632478-40000080-93bf-11ea-9ee5-86c69752a697.png">

**Comment by astrobokonon** [comment]
```
2020-05-22 17:46:42,517 ERROR    Error thrown from periodic callback:
2020-05-22 17:46:42,517 ERROR    Traceback (most recent call last):
  File "/opt/conda/lib/python3.7/site-packages/tornado/gen.py", line 501, in callback
    result_list.append(f.result())
  File "/opt/conda/lib/python3.7/site-packages/bokeh/server/session.py", line 67, in _needs_document_lock_wrapper
    result = func(self, *args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/bokeh/server/session.py", line 195, in with_document_locked
    return func(*args, **kwargs)
  File "/opt/conda/lib/python3.7/site-packages/bokeh/document/document.py", line 1164, in wrapper
    return doc._with_self_as_curdoc(invoke)
  File "/opt/conda/lib/python3.7/site-packages/bokeh/document/document.py", line 1150, in _with_self_as_curdoc
    return f()
  File "/opt/conda/lib/python3.7/site-packages/bokeh/document/document.py", line 1163, in invoke
    return f(*args, **kwargs)
  File "/home/lig/Codes/NightShift/nightshift/bokeh/ldt/ldtWeather.py", line 201, in grabNew
    mds2 = lplot.newDataCallback(cds, cols, nf, lastTimedt, y1lim)
  File "/home/lig/Codes/NightShift/nightshift/bokeh/plotting/lineplots.py", line 300, in newDataCallback
    nf.fillna(value=cfills, inplace=True)
  File "/opt/conda/lib/python3.7/site-packages/pandas/core/frame.py", line 4153, in fillna
    downcast=downcast,
  File "/opt/conda/lib/python3.7/site-packages/pandas/core/generic.py", line 6232, in fillna
    obj.fillna(v, limit=limit, inplace=True, downcast=downcast)
  File "/opt/conda/lib/python3.7/site-packages/pandas/core/series.py", line 4159, in fillna
    downcast=downcast,
  File "/opt/conda/lib/python3.7/site-packages/pandas/core/generic.py", line 6167, in fillna
    value, method = validate_fillna_kwargs(value, method)
  File "/opt/conda/lib/python3.7/site-packages/pandas/util/_validators.py", line 332, in validate_fillna_kwargs
    raise ValueError("Must specify a fill 'value' or 'method'.")
ValueError: Must specify a fill 'value' or 'method'.
```

**Comment by astrobokonon** [comment]
And it looks like MountTemp gets shredded somewhere during the data query/check; that's probably easier to fix

```
2020-05-22T17:51:46.120162726Z Checking for new data!
2020-05-22T17:51:46.120168088Z Data were updated 30.030358 seconds ago (2020-05-22 17:51:16.052399)
2020-05-22T17:51:46.120173584Z Last in CDS: 2020-05-22 17:39:19+00:00
2020-05-22T17:51:46.120179107Z Last in r  : 2020-05-22 17:49:38+00:00
2020-05-22T17:51:46.120184499Z Last in r2 : 2020-05-22 17:50:49+00:00
2020-05-22T17:51:46.120189997Z {'AirTemp': nan, 'Humidity': nan, 'DewPoint': nan, 'MountTemp': 11.673695681667, 'M': None, 'o': None, 'u': None, 'n': None, 't': None, 'T': array([None, None, None, ..., None, None, None], dtype=object), 'e': None, 'm': None, 'p': None}
2020-05-22T17:51:46.120201065Z Checking for new data!
2020-05-22T17:51:46.120206870Z Data were queried 30.043099 seconds ago (2020-05-22 17:51:16.052399)
2020-05-22T17:51:46.120212440Z Checking for new data!
2020-05-22T17:51:46.120217837Z Data were updated 30.061871 seconds ago (2020-05-22 17:51:16.052399)
2020-05-22T17:51:46.137429679Z ERROR:bokeh.util.tornado:Error thrown from periodic callback:
2020-05-22T17:51:46.137448788Z ERROR:bokeh.util.tornado:Error thrown from periodic callback:
2020-05-22T17:51:46.137792900Z ERROR:bokeh.util.tornado:Traceback (most recent call last):
```

**Comment by astrobokonon** [comment]
Attempted fix in [ligmos@dd113ff153986099ea8065e77c216c7b14453e19](https://github.com/LowellObservatory/ligmos/commit/dd113ff153986099ea8065e77c216c7b14453e19) since it is a bug introduced by the hacking last month to fix empty queries.  MountTemp isn't shredded anymore, but the plot is still screwed up.

**Comment by astrobokonon** [comment]
Closing as it's superseded by the Grafana integration(s)
